### PR TITLE
[Refacto] remove use custom code instead of account password

### PR DESCRIPTION
**Pull request by KaterinaWire**
## What's new in this PR?

Remove `useCustomCodeInsteadOfAccountPassword` because we don't use it anymore in the UI. The fallback to unlock Wire is always a custom passcode.

Needs releases with:

- [ https://github.com/wireapp/wire-ios-data-model/pull/1092 ]

### CODE REVIEW

File: Tests/Source/Data Model/ServiceUserTests.swift [hunk 1]
```diff
@@ -51,6 +51,8 @@ final class DummyServiceUser: NSObject, ServiceUser {
     
     var isVerified: Bool = false
     
+    var isTrusted: Bool = true
```
Comment (johnxnguyen) [inline Tests/Source/Data Model/ServiceUserTests.swift:54]: Is this meant to be here?

Comment (typfel) [reply Tests/Source/Data Model/ServiceUserTests.swift:54]: This is due to this major change which was released as minor https://github.com/wireapp/wire-ios-data-model/pull/1089

Comment (typfel) [reply Tests/Source/Data Model/ServiceUserTests.swift:54]: The default value should probably be false though

**Review by typfel — APPROVED** [review]

File: Tests/Source/Data Model/ServiceUserTests.swift (hunk 1 above)
Comment (KaterinaWire) [reply Tests/Source/Data Model/ServiceUserTests.swift]: Removed it and pulled changes from the develop

**Review by johnxnguyen — APPROVED** [review]

**Pull request merged**